### DISCUSSION TITLE
Database Connect Problem

### DISCUSSION THREAD

**Discussion by Aurydevelopment** [OwlGamingCommunity/V#24] [General]
My database has no password. I leave this blank and I get this error. What is the reason?

https://user-images.githubusercontent.com/64798519/141836461-69de9bff-2bae-4252-b844-10d534ca7ff5.png
https://user-images.githubusercontent.com/64798519/141836453-949897de-1454-453a-8873-b0bc8d511740.png

**Comment by okdamsam** [comment]
Create a new mysql user that uses a password and use those credentials. The MySQL connection cannot process an empty string.

**Reply by Aurydevelopment** [reply]
didn't work

![Screenshot_1](https://user-images.githubusercontent.com/64798519/145192951-b03df3cd-a187-47ab-b0a7-483011bf2b1c.png)
![Screenshot_2](https://user-images.githubusercontent.com/64798519/145192958-f0a5f888-1205-408e-9839-c00e8f69b85a.png)

**Reply by okdamsam** [reply]
Then you're doing something wrong.

**Reply by Aurydevelopment** [reply]
I am using XAMPP. Is that why I'm having error?

**Reply by okdamsam** [reply]
> I am using XAMPP. Is that why I'm having error?

No. XAMPP works fine for me.

**Comment by ragesawyer** [comment]
i'm having the same problem, the problem is most likely that there is more than one "GAME_DATABASE_PASSWORD", so I'll make the password "admin" and test it.

**Reply by Aurydevelopment** [reply]
didn't work
